### PR TITLE
rtss: Add version 7.2.3

### PR DESCRIPTION
**Pull request by r15ch13**
- Add standalone version of RivaTuner Statistics Server
- Migrates profiles from old MSI Afterburner persist structure

### CODE REVIEW

File: bucket/rtss.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "url": "https://download-eu2.guru3d.com/rtss/%5BGuru3D.com%5D-RTSSSetup723Build20686.zip#/sourceforge.net.zip",
+    "hash": "f29e2d2b68b4c54d2172739a35e666979287a5966058af7b585e808a6e8b77d3",
+    "suggest": {
+        "Visual C++ Redist 2008": "extras/vcredist2008",
+        "MSI Afterburner": "extras/msiafterburner"
+    },
+    "persist": "Profiles",
+    "bin": "RTSS.exe",
+    "pre_install": [
+        "Expand-7zipArchive \"$dir\\RTSSSetup*.exe\" \"$dir\" -Removal",
```
Comment (Ash258) [inline bucket/rtss.json]:
```suggestion
        "Expand-7zipArchive \"$dir\\RTSSSetup*.exe\" -Removal",
```

File: bucket/rtss.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        "Get-ChildItem \"$dir\" -Recurse -Include @('Guru3D.com', '$PLUGINSDIR', 'Uninstall.exe.nsis', '$R0') | Remove-Item -Recurse",
```
Comment (Ash258) [inline bucket/rtss.json]:
```suggestion
        "Remove-Item \"$dir\\*\" -Include @('Guru3D.com', '$PLUGINSDIR', 'Uninstall.exe.nsis', '$R0') -Recurse",
```

File: bucket/rtss.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        "Move-Item \"$dir\\RTSSHooks.dll.copy\" \"$dir\\RTSSHooks.dll\"",
+        "Move-Item \"$dir\\RTSSHooks64.dll.copy\" \"$dir\\RTSSHooks64.dll\""
+    ],
+    "post_install": [
+        "$rtssProfiles = \"$persist_dir\\..\\msiafterburner\\RTSS_Profiles\"",
+        "if(Test-Path \"$rtssProfiles\") {",
+        "    warn 'Migrating RTSS profiles...'",
+        "    Move-Item -Path \"$rtssProfiles\\*\" -Destination \"$persist_dir\\Profiles\\\"",
```
Comment (Ash258) [inline bucket/rtss.json]: Trailing slash is needed?

File: bucket/rtss.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        "    Remove-Item \"$rtssProfiles\"",
```
Comment (Ash258) [inline bucket/rtss.json]:
```suggestion
        "    Remove-Item \"$rtssProfiles\" -Recurse",
```

File: bucket/rtss.json (hunk 2 above)
Comment (linsui) [reply bucket/rtss.json]: This script can't remove $PLUGINSDIR

Comment (r15ch13) [reply bucket/rtss.json]: image
Can't reproduce. 🤔

Comment (linsui) [reply bucket/rtss.json]: The same in pscore, Guru.com is not removed, too.

2019-09-23 21_02_27-Windows PowerShell
2019-09-23 21_05_38-65427368-74b04f00-de12-11e9-9e53-fde0a8bc4890 png（PNG 图像，1124x601 像素） - 缩放 (65%)

Comment (linsui) [reply bucket/rtss.json]: I got it... The -Recurse not work.

Comment (r15ch13) [reply bucket/rtss.json]: Hm, it doesn't work if the directory contains files.

Comment (r15ch13) [reply bucket/rtss.json]: It's known issue: image
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/remove-item?view=powershell-6

Comment (r15ch13) [reply bucket/rtss.json]: `Remove-Item 'D:\scoop\apps\rtss\7.2.3\$PLUGINSDIR' -Recurse` works fine. I guess it's a problem with the wildcard/include. Will change it back to `gci`

Comment (Ash258) [reply bucket/rtss.json]: Sorry. My bad i forgot to check if fix was backported to any of older powershell versions.

Comment (r15ch13) [reply bucket/rtss.json]: fixed with https://github.com/lukesampson/scoop-extras/commit/c6739bed7437517e8c6238f8399d054a716a2c21 and https://github.com/lukesampson/scoop-extras/commit/90f09fd7470f2c8ba13ed84f205cfbcac6da581d

**Pull request merged**